src/components/OSCALMetadata.js

```diff
@@ -38,17 +53,174 @@ const OSCALMetadataPartiesCard = styled(Paper)(({ theme }) => ({
   backgroundColor: theme.palette.background.paper,
   position: "relative",
   overflow: "auto",
-  maxHeight: "12em",
+  maxHeight: "18em",
 }));
 
 // Returns a string with a locality-sensitive representation of this date
 const formatDate = (isoDate) => new Date(isoDate).toLocaleString();
 
+const TELEPHONE_ICON_MAPPING = {
+  home: <HomeIcon />,
+  mobile: <SmartphoneIcon />,
+  office: <BusinessIcon />,
+};
+
+const ADDRESS_ICON_MAPPING = {
+  home: <HomeIcon />,
+  work: <BusinessIcon />,
+};
+
+const UNKNOWN_TYPE_ICON = <HelpOutlineIcon />;
+
+function getPhoneIcon(contactType) {
+  return TELEPHONE_ICON_MAPPING[contactType] ?? UNKNOWN_TYPE_ICON;
+}
+
+function getAddressIcon(contactType) {
+  return ADDRESS_ICON_MAPPING[contactType] ?? UNKNOWN_TYPE_ICON;
+}
```

Comment:
```suggestion
function getIconOfType(mapping, contactType) {
  if (!mapping || !contactType) {
    return UNKNOWN_TYPE_ICON;
  }
  return mapping[contactType] ?? UNKNOWN_TYPE_ICON;
}

function getPhoneIcon(contactType) {
  return getIconOfType(TELEPHONE_ICON_MAPPING, contactType);
}

function getAddressIcon(contactType) {
  return getIconOfType(ADDRESS_ICON_MAPPING, contactType);
}
```

Realizing that the `type` can be `null`/`undefined`, we probably want to harden this just a bit. Undecided on whether it's cleanest to extract the logic to a generic helper or not. But we at least want to handle `contactType` being falsey.